### DISCUSSION TITLE
is boilTemp the correct name?

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#115] [General]
In the Boiler, telegram 0x19/UBAMonitorSlow has the `boilTemp` starting at offset 2. I'd like to change the name as just calling it 'Boiler Temperature' is a bit ambiguous. Looking at Norbert's HT3 notes he calls it "**Maximale Temperatur**" (German) and emswiki.thefischer.net calls it "**Kessel-Ist-Temperatu**r".  I'm not really sure what it is. 

Should we call it 

a) Maximun Temperature, or
b) Actual Temperature

@MichaelDvP ?

**Comment by MichaelDvP** [comment]
It's the measured actual temperature of the water inside the boiler (vessel), mainly the same as currflowtemp (from 0x18, published more often). "boiler" is ambiguous because it describes the whole heating system and it is the metal vessel holding the water inside (german: Kessel). 
I think it's the right name, or change `boil` in boilTemp, boilHystOn, boilHystOff to `vessel`?

**Reply by proddy** [reply]
Thanks Michael. Reason I was asking is how Home Assistant creates the entities. To make them Unique I prefix the device type, so for boilTemp it looks like:

<img width="402" alt="Screen Shot 2021-09-19 at 08 43 03" src="https://user-images.githubusercontent.com/1230712/133918316-767593f3-d5da-4506-b342-e1a6f4524903.png">

which is kinda stupid with the "Boiler boiler..." duplication.

**Comment by proddy** [comment]
I renamed 'boiltemp'

```C
MAKE_PSTR_LIST(boilTemp, F("boiltemp"), F("actual boiler temperature"))
```
